docs/main/faqs/faqs.md

```diff
@@ -648,6 +648,18 @@ Now that `0000:01:00.0` is bound to `vfio-pcie`, you can create your pools with
 
 [Go to top](#top)
 
+### How do I install Replicated PV Mayastor on a Kubernetes cluster with a custom domain?
+
+You must configure etcd for Replicated PV Mayastor during installation if the domain name of your Kubernetes cluster is not 'cluster.local'.
```

Comment:
"You must configure etcd" seems incorrect.
Need to be rephrased differently

Reply:
Re-writing as:
If the domain name of your Kubernetes cluster is not 'cluster.local', you have to specify the custom domain during installation as follows:"